manim/utils/tex_file_writing.py

```diff
@@ -177,12 +195,10 @@ def compile_tex(tex_file, tex_compiler, output_format):
                 if error_pos:
                     with open(tex_file, "r") as g:
                         tex = g.readlines()
-                        logger.error("LaTeX compilation error! LaTeX reports:")
+                        logger.error("LaTeX compilation error: {log[error_pos][2:])}")
```

Comment:
Was this meant to be an f-string? Currently it just prints `LaTeX compilation error: {log[error_pos][2:])}`. Seems to have an extra parenthesis as well. Also, making this an f-string causes an error since `error_pos` is a list of integers. @clarfonthey could you please explain what was the rationale behind this line?

Reply:
Oh...thanks for pointing that out. I wonder how I could miss that when I tested it locally.